src/web/module/history/component/View.tsx

```diff
@@ -84,10 +84,11 @@ const View = (): JSX.Element => {
       {data &&
         data.map((item: History) => {
           return (
-            <>
+            // @ts-expect-error expected type
+            <div key={currentDate ? currentDate : Math.random()}>
```

Comment:
```suggestion
            <div key={currentDate || Math.random()}>
```

Reply:
updated in https://github.com/RR-Ardhika/murojaah/pull/31/commits/7fc990252e421b767e5498afd446e2345d775745, thank you